### ISSUE TITLE
Shopping list

### ISSUE THREAD

**Issue by ghost** [abersailbot/kitty-hardware#11]
- [x] M3 Nylon screws, bolts, nuts
- [ ] Bungee cord elastic stuff (The stuff that's on Dewi)
- [ ] marine enamel paint
- [x] Vaseline/grease
- [x] O-ring 220x5 mm (x2)
- [x] O-ring 115x5 mm
- [x] MMCX to SMA pigtail

**Comment by kragniz** [comment]
Spray paint for the hull.

**Comment by kragniz** [comment]
I just updated the issue to use checkboxes.

**Comment by ghost** [comment]
Paint for #12, what colour?

**Comment by naturesyouth** [comment]
is spray paint the best choice?

**Comment by LukeILunn** [comment]
O-ring, for front hatch, need to check dimensions. Also vaseline.

**Comment by Danyc0** [comment]
@naturesyouth: yea, spray paint will be good. @JaffaCakes6: Kitty is already mostly yellow, I'd guess she just needs more of that

**Comment by ghost** [comment]
Updated to include more items.

**Comment by naturesyouth** [comment]
Are you guys going sand and reprime kitty?

**Comment by ghost** [comment]
Undecided. At this point I think we're just going for seafaring paint rather than pretty paint, but I'm not sure.

Also add more items, and purchasing some too.

**Comment by NicholasRostant** [comment]
If you're hunting for seafaring paint Marine Enamel is probably what you want.

**Comment by TomGC96** [comment]
Yes certainly some sort of marine paint, not any old paint - it'll peel off like crazy. Perhaps some sort of gelcoat spray?

**Comment by NicholasRostant** [comment]
Gelcoat would probably be way too heavy in the kind of thicknesses it's usually applied. I'd recommend something along the lines of [this stuff](http://www.pettitpaint.com/product.asp?id=274), which I've used before, its topside paint so should be more than durable enough.
